conditions.py

```diff
@@ -0,0 +1,34 @@
+# While - repeats a body of code as long as condition is true
+
+counter = 1
+while counter <= 5:
+    print("Hello, World")
+    counter = counter + 1
+
+# if BOOLEAN EXPRESSION:
+#     STATEMENTS
+food ='chapati'
+if food == 'chapati':
+    print('My favourite!')
+
+# if-else STATEMENT
+
+food ='chips'
+if food == 'chapati':
```

Comment:
You can do strict typing comparison with 3 equal signs i.e `===`

Reply:
Noted